test/functions/operatorOverloads/operatorMethods/lifetimeOnMethod.chpl

```diff
@@ -0,0 +1,21 @@
+record Foo {
+  var x: int;
+
+  operator <(lhs: Foo, rhs: Foo) {
+    return lhs.x < rhs.x;
+  }
+
+  proc otherMethod(arg1: int) lifetime this < arg1 {
```

Comment:
This is sortof moot because `int` doesn't have a lifetime. You'd have to make some `borrowed class` instances for it to really do something meaningful. Nonetheless it is syntactically reasonable and if this didn't work before and now does it's a reasonable test.

Reply:
I've updated the test I think how you were suggesting.  Does that look right?

Reply:
yes it is more reasonable, thanks

Reply:
to really do something with the lifetime checker we need the record to have a borrowed class field. But that is not the point of this test.

Reply:
Thanks :)